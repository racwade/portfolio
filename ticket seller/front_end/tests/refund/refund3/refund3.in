login
test_admin
refund
test_buy
test_not
100
logout