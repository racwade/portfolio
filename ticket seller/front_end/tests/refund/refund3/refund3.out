welcome to the ticket seller system
enter username:
logged in as test_admin
enter event name:
enter buyer name:
enter seller name:
enter refund amount:
error: seller name not found in system
bye